### PR TITLE
Create how-to-release.md

### PR DESCRIPTION
**Pull request by jbusecke**
- [ ] Closes #xxxx
 - [ ] Tests added
 - [ ] Passes `black . `
 - [ ] User visible changes (including notable bug fixes) are documented in `whats-new.rst`
 - [ ] New functions/methods are listed in `api.rst`

### CODE REVIEW

**Comment by jbusecke** [comment]
@rabernat for the next time, so everyone else can release without needing my help!

File: doc/how-to-release.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# How to release a new version of xgcm (for maintainers only)
+The process of releasing at this point is very easy. 
+
+We need only two things: A doc PR and and making a release on github.
+
+1. Make sure that all the new features/bugfixes etc are appropriately documented in `doc/whats-new.rst`, add the date to the current release and make an empty (unreleased) entry for the next minor release as a PR.
+2. Navigate to the 'tags' symbol on the repos main page, click on 'Releases' and on 'Draft new release' on the right.
+  - Add the versionnumber and a short description and save the release.
```
Comment (navidcy) [inline doc/how-to-release.md:8]: versionnumber  -> version number
?

**Comment by timothyas** [comment]
Thank you for this @jbusecke! This is super useful.

**Comment by jbusecke** [comment]
Should we actually link it in the docs?

File: doc/how-to-release.md (hunk 1 above)
Comment (dcherian) [inline doc/how-to-release.md:8]:
```suggestion
  - Add the version number and a short description and save the release.
```

**Comment by dcherian** [comment]
> Should we actually link it in the docs?

I think so. Doesn't hurt...

File: doc/how-to-release.md (hunk 1 above)
Comment (dcherian) [reply doc/how-to-release.md]: Thanks!

**Comment by dcherian** [comment]
I don't see it?

Seems like it could go under contributor guide

image

**Review by dcherian — APPROVED** [review]
Thanks!

**Pull request merged**